README.md

```diff
@@ -1,2 +1,84 @@
 # rokt-demo-ios
-Rokt iOS Demo Application
+Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
```

Comment:
Who use? 

To whom this app distributes to?

Reply:
> let's assume that the expected readers for this README are not us but grad engineers in NYC. They should be able to understand what this repo is for, able to debug on local, fix a small bug, test and deploy it to prod.

I think the user of this repository, should at least have basic iOS knowledge if they want to test and deploy it to prod. The professional level of this readme would be reduced a lot as we are explaining every single terminology like MVVM. But I would reflect all the changes you mentioned here.

Reply:
Updated entirely